src/ui/index.js

```diff
@@ -29,7 +29,7 @@ const scripts = () => {
 	return js(filePath, {
 		optimize: config.isDevelopmentMode === false,
 		nodeGlobals: config.isDevelopmentMode === true,
-		replace: { 'process.env.NODE_ENV': JSON.stringify(config.isDevelopmentMode === true ? 'development' : 'production') },
+		replace: { 'process.env.NODE_ENV': JSON.stringify(config.isDevelopmentMode === true ? 'development' : 'production'), 'process.env.BASE_URL': config.baseUrl },
```

Comment:
You can move this up to the index function. The last argument of the layout function accepts environment variables that you can access via `window.env` in the UI scripts.